src/tests/rsa.rs

```diff
@@ -415,7 +415,7 @@ fn test_rsa_operations() {
     );
     assert_eq!(ret, CKR_OK);
 
-    #[cfg(not(feature = "fips"))]
+    #[cfg(feature = "dynamic")]
```

Comment:
looking at these changes, its now correct, but completely unreadable why this is enabled only in the dynamic build so I would prefer a comment about what is discussed #181 and in the previous comments why this works only in dynamic builds.
Likely something like in the signatures test would do:
```
    /* SHA1 is disabled in the openssl submodule */
```

Reply:
Yeah, I thought the same and then forgot to put the comment, good catch!